components/BlurhashCanvas.vue

```diff
@@ -37,16 +50,27 @@ export default Vue.extend({
   },
   methods: {
     draw() {
-      const pixels = decode(this.hash, this.width, this.height);
-      if (pixels) {
-        const ctx = (this.$refs.canvas as HTMLCanvasElement).getContext('2d');
-        const imageData = ctx?.createImageData(this.width, this.height);
-        if (imageData) {
-          imageData.data.set(pixels);
-          ctx?.putImageData(imageData, 0, 0);
+      try {
+        const pixels = decode(this.hash, this.width, this.height);
```

Comment:
We can use [isBlurhashValid](https://github.com/woltapp/blurhash/tree/master/TypeScript#isblurhashvalidblurhash-string---result-boolean-errorreason-string-) to check the hash before.

We could probably just check before calling draw() and change `validHash` to false (And change it back to true if the hash is valid, to keep reactivity in case the hash changes)

Reply:
Good idea, Updated